hack/generator/pkg/codegen/pipeline/create_arm_types.go

```diff
@@ -15,12 +15,12 @@ import (
 	"github.com/Azure/azure-service-operator/hack/generator/pkg/astmodel"
 )
 
-// createARMTypes walks the type graph and builds new types for communicating
+// CreateARMTypes walks the type graph and builds new types for communicating
 // with ARM
-func createARMTypes(idFactory astmodel.IdentifierFactory) PipelineStage {
-	return MakePipelineStage(
+func CreateARMTypes(idFactory astmodel.IdentifierFactory) Stage {
+	return MakeStage(
 		"createArmTypes",
-		"Creates ARM types",
+		"Create custom types for interaction with ARM",
```

Comment:
minor:
```suggestion
		"Create types for interaction with ARM",
```

Unsure what custom is alluding to here but I don't think these types are more custom than any of our other types?